tldw_Server_API/app/core/Utils/executor_registry.py

```
@@ -62,9 +67,16 @@ async def shutdown_executor(name: str, wait: bool = True, cancel_futures: bool =
 
 
 async def shutdown_all_registered_executors(wait: bool = True, cancel_futures: bool = True) -> None:
```

Ruff (0.15.7)

[warning] Boolean-typed positional argument in function definition

(FBT001)